### PR TITLE
Add liac-arff

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

_Created with Grayskull_

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge/help-python Would be ready for review, thanks!

File: recipes/liac-arff/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 3220d0af6487c5aa71b47579be7ad1d94f3849ff1e224af3bf05ad49a0b5c4da
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - ez_setup
```
Comment (BastianZim) [inline recipes/liac-arff/meta.yaml:20]: I'm not completely sure about this requirement but it was suggested by Grayskull.

The setup.py file first imports setuptools and then falls back to ez_setup.
See: https://github.com/renatopp/liac-arff/blob/35816f5c393a6a04ed6fcc2d41b0309433fa4cd3/setup.py#L10-L17

Comment (chrisburr) [reply recipes/liac-arff/meta.yaml:20]: I'd be inclined to drop it:
```suggestion
```

File: recipes/liac-arff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  summary: A module for read and write ARFF files in Python.
+  doc_url: https://pythonhosted.org/liac-arff/
+  dev_url: https://github.com/renatopp/liac-arff
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  description: |
+    The liac-arff module implements functions to read and write ARFF files in Python. It was created in the Connectionist Artificial Intelligence Laboratory (LIAC), which takes place at the Federal University of Rio Grande do Sul (UFRGS), in Brazil.
+
+    ARFF (Attribute-Relation File Format) is an file format specially created for describe datasets which are used commonly for machine learning experiments and softwares. This file format was created to be used in Weka, the best representative software for machine learning automated experiments.
```
Comment (chrisburr) [inline recipes/liac-arff/meta.yaml:45]:
```suggestion
    The liac-arff module implements functions to read and write ARFF files in
    Python. It was created in the Connectionist Artificial Intelligence
    Laboratory (LIAC), which takes place at the Federal University of Rio Grande
    do Sul (UFRGS), in Brazil.

    ARFF (Attribute-Relation File Format) is an file format specially created
    for describe datasets which are used commonly for machine learning
    experiments and softwares. This file format was created to be used in Weka,
    the best representative software for machine learning automated experiments.
```

Comment (BastianZim) [reply recipes/liac-arff/meta.yaml:44]: Thank you!

**Pull request merged**